### ISSUE TITLE
Can't identify hmm sources common to multiple genomes

### ISSUE THREAD

**Issue by dcourtine** [merenlab/anvio#816]
Hi Meren!
I was interested in the phylogenomics pipeline and I ran into an error when I wanted to check HMM sources common to multiple genomes.
I got this message:
```
courtine$ anvi-get-sequences-for-hmm-hits --external-genomes external-genomes.txt --list-hmm-sources

HMM SOURCES COMMON TO ALL 52 GENOMES
===============================================
Traceback (most recent call last):
  File "/Users/courtine/GitHub/anvio/bin/anvi-get-sequences-for-hmm-hits", line 280, in <module>
    main(args)
  File "/Users/courtine/GitHub/anvio/bin/anvi-get-sequences-for-hmm-hits", line 101, in main
    s = hmmopswrapper.SequencesForHMMHitsWrapperForMultipleContigs(args, hmm_sources)
  File "/Users/courtine/GitHub/anvio/anvio/hmmopswrapper.py", line 33, in __init__
    self.load_genomes_descriptions(skip_functions=True, init=False)
  File "/Users/courtine/GitHub/anvio/anvio/genomedescriptions.py", line 200, in load_genomes_descriptions
    self.sanity_check()
  File "/Users/courtine/GitHub/anvio/anvio/genomedescriptions.py", line 404, in sanity_check
    self.list
```
I also have a similar error when I wanted to list common genes:

```
courtine$ anvi-get-sequences-for-hmm-hits --external-genomes external-genomes.txt --hmm-source Rinke_et_al --list-available-gene-names

GENES IN HMM SOURCES COMMON TO ALL 52 GENOMES
===============================================
Traceback (most recent call last):
  File "/Users/courtine/GitHub/anvio/bin/anvi-get-sequences-for-hmm-hits", line 280, in <module>
    main(args)
  File "/Users/courtine/GitHub/anvio/bin/anvi-get-sequences-for-hmm-hits", line 101, in main
    s = hmmopswrapper.SequencesForHMMHitsWrapperForMultipleContigs(args, hmm_sources)
  File "/Users/courtine/GitHub/anvio/anvio/hmmopswrapper.py", line 33, in __init__
    self.load_genomes_descriptions(skip_functions=True, init=False)
  File "/Users/courtine/GitHub/anvio/anvio/genomedescriptions.py", line 200, in load_genomes_descriptions
    self.sanity_check()
  File "/Users/courtine/GitHub/anvio/anvio/genomedescriptions.py", line 404, in sanity_check
    self.list_HMM_info_and_quit()
  File "/Users/courtine/GitHub/anvio/anvio/genomedescriptions.py", line 102, in list_HMM_info_and_quit
    s = list(self.genomes.values())[0]['hmm_sources_info'][source]
KeyError: 'hmm_sources_info'
```
I made sure that the 52 databases contain HMM hits. I tried to extract the concatenation of `Ribosomal_L1,Ribosomal_L2` from *Rinke_et_al* and it worked.

I don't really need this feature, my genomes are 100% complete with *Rinke_et_al* dataset, it is just for debugging purpose 😄 

Best,
Damien

**Comment by meren** [comment]
Hey Damien,

Is this `master`?

@ozcan, does this look familiar to you?


Best,

**Comment by dcourtine** [comment]
Yes of course!
I did a `git pull` when I got this error. Bu the error was still here :/

Damien

**Comment by meren** [comment]
Thanks, Damien. We will look into this.

**Comment by meren** [comment]
It is now fixed by 2f794faf6a8d8b2238930aa7e15625d2a4da5a61. Thanks, Damien.
